README.md

```diff
@@ -14,20 +14,21 @@ On PC, you can play Pixel Wheels alone or with a friend.
 
 Great! For Linux, macOS and Windows you can get binaries from [Pixel Wheels page on itch.io][itch].
 
-_ARM-based Mac users: Note that Pixel Wheels does not run on ARM-based Macs for now. This will be fixed when [libgdx ARM-based Mac support][libgdx-arm-mac] is ready._
+_ARM-based Mac users: Note that Pixel Wheels (except for special build) does not run on ARM-based Macs for now. This will be added when [performance issues with libGDX][libgdx-arm-mac] will be resolved. You can download game with ARM support for testing at [this][version-to-test] link._
```

Comment:
This is not correct: this build is an Intel Mac build. I just do not have access to a Mac anymore, so I asked around for someone with an Intel Mac to test it.

Reply:
I will keep "performance issues with libGDX", because it is the main reason to not update libGDX for now, plus libGDX 1.11.0 finally supports ARM Macs.

Reply:
Please don't, it's just going to confuse players: they should not care about details like libgdx versions.

On the other hand, it's good to know that 1.11.0 is out, but I won't update to it for 0.23.0. I want to release 0.23.0 soon, and I prefer updating libgdx early in the development cycle of a new Pixel Wheels version, to have more time to catch regressions.

Reply:
I think we can't leave this without changing, because ARM support is already done and users will be misleaded by keeping this as is (issue is closed, linked PR is merged, support is added). Maybe just say, that "this will be added later"?

Reply:
ARM support is not done as far as Pixel Wheels is concerned.

It's in libgdx 1.11.0 which is not the version which will be shipped with Pixel Wheels 0.23.0. It **may** be the version shipped with Pixel Wheels 0.24.0, if 1.10.0 performance issues are fixed.

Reply:
I changed my mind. Now that 1.11.0 is out, you are right, the message needs to change. I think something like this would be good:

> _ARM-based Mac users: Note that Pixel Wheels does not run on ARM-based Macs for now. This will be added when the game is updated to use libGDX 1.11.0._

Reply:
Done.